webapp/app/landing/Introduction/AreaSelector/AreaSelector.js

```diff
@@ -0,0 +1,59 @@
+import './areaSelector.less'
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import axios from 'axios'
+
+import { Area, Country } from '@common/country'
+import * as BasePaths from '@webapp/main/basePaths'
+
+import { useI18n } from '@webapp/components/hooks'
+
+import DropdownAreas from './DropdownAreas'
+
+const areas = {
```

Comment:
Should this be in common/country/area?

Reply:
This is local state . The keys of the dropdown